bin/dlang/codecov

```diff
@@ -42,9 +42,22 @@ git archive --format=tar HEAD | (cd "$tmp" && tar xf -)
 cp -a .git "$tmp/"
 cp -av "$r/bin/codecov-bash" "$tmp/codecov"
 
+# Build codecov flags
+flags=
+for flag in "$F" "$DIST" "dmd_$DMD"
+do
+    # Codecov only accepts flags that match ^[\w\,]+$
+    flag="$(echo "$flag" | sed -r 's/\W/_/g')"
+    if test -z "$flags"
+    then
+        flags="-F $flag"
```

Comment:
Protection against empty `$flag` while you are at it? :)

Reply:
Ping @leandro-lucarella-sociomantic

Reply:
Mmm, I guess, I wanted to add a prefix to all flags, like we have with dmd. Maybe I can do that so flags will show if it was built with an empty `$F` for example.